lib/main.sh

```diff
@@ -395,7 +395,7 @@ LINUXSOURCEDIR="${KERNELDIR}/$(branch2dir "${KERNELBRANCH}")"
 BSP_CLI_PACKAGE_NAME="armbian-bsp-cli-${BOARD}"
 BSP_CLI_PACKAGE_FULLNAME="${BSP_CLI_PACKAGE_NAME}_${REVISION}_${ARCH}"
 
-CHOSEN_UBOOT=linux-u-boot-${BRANCH}-${BOARD}
+CHOSEN_UBOOT=linux-u-boot-${BOARD}-${BRANCH}
```

Comment:
Sure this is correct naming? BTW. Its planned to be changed once this year as we [did it with BSP package](https://github.com/armbian/build/pull/2779).

Reply:
Maybe not, but package name is currently incompatible with deb filename.
IMO we have to change:
a) DEB filename as package name 
or 
b) package name in DEB specfile as DEB filename.

Reply:
yes, naming should be adjusted properly. I plan to deal with this problem once https://forum.armbian.com/topic/13949-creating-packages-in-the-armbian-build-system/ gets to the merge request state. So we don't scatter those changes around too much.